config/exchanges.json

```diff
@@ -1544,15 +1551,16 @@
   "NO-NO4->RU-1": {
     "lonlat": [30.1437, 69.52],
     "parsers": {
-      "exchange": "statnett.fetch_exchange"
+      "exchange": "ENTSOE.fetch_exchange",
+      "exchangeForecast": "ENTSOE.fetch_exchange_forecast"
     },
     "rotation": 120
   },
```

Comment:
I did a final check and noticed this exchange is not in ENTSO-E at all. 
Curiously enough it is always zero at Statnett as well?

Should we just remove it?

Reply:
Looks like data on Statnett is 0 since 2020/10/24 and before that we have small values. 

it makes sense that the exchanges are 0 at the moment as all flows are stopped. And from what I've read in this article it's also normal that the values are so small. (https://thebarentsobserver.com/en/industry-and-energy/2022/04/nordic-grid-cuts-russian-power)

I think we should unarchive the parser and use it just for the exchange and monitor

Reply:
sounds good!